pyproject.toml

```
@@ -1,3 +1,45 @@
+[tool.poetry]
+name = "sceptre-request-resolver"
+version = "0.0.2"
+packages = [{ include = "resolver" }]
+readme = "README.md"
+homepage = "https://github.com/Sceptre/sceptre-request-resolver"
+repository = "https://github.com/Sceptre/sceptre-request-resolver"
+authors = ["Khai Do <zaro0508@gmail.com>"]
+description = "A Sceptre resolver to make requests from REST API endpoints"
+keywords = ["sceptre", "sceptre-resolver"]
+license = "Apache2"
```

```suggestion
license = "Apache-2.0"
```
https://python-poetry.org/docs/pyproject/#license

---

fixed